accounts/views.py

```diff
@@ -32,4 +32,21 @@ def post(self, request):
             )
             return JsonResponse ({"MESSAGE":"SUCCESS"}, status = 201)
         except KeyError:
-            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
\ No newline at end of file
+            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
+
+class SigninView(View):
+    def get(self, request):
+        data = json.loads(request.body)
+        name = Account.objects.get(email=data["email"])
+
+        try:
+            if bcrypt.checkpw(data["password"].encode("utf-8"), name.password.encode("utf-8")):
+                access_token = jwt.encode({"name":name.pk}, SECRET_KEY, algorithm = 'HS256')
```

Comment:
```suggestion
                access_token = jwt.encode({"user_id":user.id}, SECRET_KEY, algorithm = 'HS256')
```
name이라는 변수보다는 user가 더 어울리는 변수입니다.

Reply:
넵 수정했습니다